Components/Motors/SongMC.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using Scarlet.Filters;
+using Scarlet.IO;
+using Scarlet.Utilities;
+using System.Threading;
+using Scarlet.Components;
+
+namespace Song
+{
+    public class SongMC : IMotor

```

Comment:
Is this actually what the motor controller is called? I'd give it its proper name so that down the line people don't get confused about which motor controller was used on songbird.